### ISSUE TITLE
How could I add tensorboad visualization?

### ISSUE THREAD

**Issue by eugtanchik** [thtrieu/darkflow#94]
Is there any way to add tensorboard visualization? I need to report loss and loss_mva values via tensorboard. Where in the original code should I start from?

**Comment by Prakash19921206** [comment]
i think it would be amazing to check trained model, if someone can integrate tf.summary in training code

**Comment by dontfollowmeimcrazy** [comment]
Anyone working on this? I would like to contribute!

**Comment by dontfollowmeimcrazy** [comment]
I added the summaries for yolo and yolov2 loss in PR #150. Need some more days for the loss_mva.

**Comment by beebrain** [comment]
@dontfollowmeimcrazy  Thank you very much, I am training and looking your tensorboard. It work.
![image](https://cloud.githubusercontent.com/assets/10016227/24835160/f11d716a-1d23-11e7-9a59-b4bab1bfb151.png)
